sphinx/source/developers/consensus.rst

```diff
@@ -24,6 +28,12 @@ There is an option of enabling CCF with PBFT as a consensus protocol providing B
 
 For more information on the PBFT protocol please see the original `PBFT paper <http://pmg.csail.mit.edu/papers/osdi99.pdf>`_.
 
+PBFT parameters can be configured when starting up a network (see :ref:`here <operators/start_network:Starting a New Network>`). The paramters that can be set via the cli are:
+
+- ``pbft-view-change-timeout-ms`` is the PBFT view change timeout in milliseconds. If a backup does not receive the pre-prepare message for a request forwarded to the primary after this timeout, the backup triggers a new view change.
+- ``pbft-status-interval-ms`` is the PBFT status timer interval in milliseconds. All PBFT nodes send messages containing their status to all other known nodes at regular intervals defined by this timer interval.
```

Comment:
I think we should make clear what status means here and what the consequences are for setting a small/large value.

Reply:
I think that is not that straightforward to do without going into PBFT detail which we have agreed to not do in this section and therefore linked the PBFT paper for users to get familiar with each consensus, no?